### PR TITLE
Version bump and badges

### PR DESCRIPTION
**Pull request by ghost**
On mobile, so hard to check everything. @ayyaruq - please review before merging. Any suggestions will have to wait for a bit, though.

### CODE REVIEW

**Comment by ayyaruq** [comment]
I'd add badges for the package.json version, license, and maybe some other shit. Also standardise the styles on prob flat or plastic. Check out shields.io for a single place to do those. Could also think about open issues/PRs, or maybe a xivapi discord link.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+[![](https://data.jsdelivr.com/v1/package/npm/vue-xivtooltips/badge)](https://www.jsdelivr.com/package/npm/vue-xivtooltips) ![Test](https://github.com/nonowazu/vue-xivtooltips/workflows/Test/badge.svg)
```
Comment (ayyaruq) [inline README.md]: I'd break this up, into new lines, like:

```suggestion
[![Latest Version](https://img.shields.io/github/package-json/v/nonowazu/vue-xivtooltips?style=plastic)]
[![CDN](https://data.jsdelivr.com/v1/package/npm/vue-xivtooltips/badge?style=rounded)](https://www.jsdelivr.com/package/npm/vue-xivtooltips)
[![Test](https://github.com/nonowazu/vue-xivtooltips/workflows/Test/badge.svg)]
[![Open Pull Requests](https://img.shields.io/github/issues-pr/nonowazu/vue-xivtooltips?style=plastic)](https://github.com/nonowazu/vue-xivtooltips/pulls)
[![Open Issues](https://img.shields.io/github/issues/nonowazu/vue-xivtooltips?style=plastic)](https://github.com/nonowazu/vue-xivtooltips/issues)
[![Licence](https://img.shields.io/github/license/nonowazu/vue-xivtooltips?style=plastic)](https://github.com/nonowazu/vue-xivtooltips/blob/master/LICENSE)
```

**Review by ayyaruq — APPROVED** [review]
Basically adding a few badges. Approved, but maybe apply the suggestion. Using the plastic theme, flat might be better, or social to make everything look like jsdelivr.

**Comment by ghost** [comment]
should be good to go

**Pull request merged**